c/tskit/tables.c

```diff
@@ -8920,7 +8920,7 @@ simplifier_record_edge(simplifier_t *self, double left, double right, tsk_id_t c
     interval_list_t *tail, *x;
     bool skip;
 
-    if (!!(self->options & TSK_SIMPLIFY_REDUCE_TO_SITE_TOPOLOGY)) {
```

Comment:
In case anyone is wondering, I got rid of a bunch of these "not not"s because they're not needed from the C perspective and they don't make the code any easier to read.